_dev/src/store/modules/product-feed/actions.ts

```diff
@@ -31,7 +31,7 @@ import Categories from '@/enums/product-feed/attribute-mapping-categories';
 import {runIf} from '../../../utils/Promise';
 import DeliveryType from '../../../enums/product-feed/delivery-type';
 import {ShippingSetupOption} from '@/enums/product-feed/shipping';
-import {generateCustomCarrier} from '@/providers/shipping-rate-provider';
+import {fromApi, generateCustomCarrier, toApi} from '@/providers/shipping-rate-provider';
```

Comment:
Should generateCustomCarrier be removed from the project?

Reply:
I'm gonna delete it on actions, after that I'm gonna make some unit test for test new methods